nemo/collections/asr/audio_preprocessing.py

```diff
@@ -738,6 +742,163 @@ def output_ports(self):
         }
 
 
+class TimeStretchAugmentation(NonTrainableNM):
+    def __init__(
+        self,
+        sample_rate: int,
+        probability: float,
+        min_speed_rate: float = 0.9,
+        max_speed_rate: float = 1.1,
+        num_rates: int = 5,
+        n_fft: int = 512,
+    ):
+        """
+        Time-stretch an audio series by a fixed rate while preserving pitch.
```

Comment:
This time-stretches a batch of audio signals not just 1 audio signal. I also assume that all elements in the batch is stretched by the same amount?

Reply:
Yes it performs one time stretch (randomly sampled) on all items of that batch. I'll make it clear in the docstring.

Reply:
Added note to docstring